addons/backgrounds/src/containers/BackgroundSelector.tsx

```diff
@@ -131,8 +131,8 @@ export const BackgroundSelector: FunctionComponent = memo(() => {
       >
         <IconButton
           key="background"
-          title="Change the background of the preview"
-          active={selectedBackgroundColor !== 'transparent'}
+          title="Change the theme of the preview"
```

Comment:
I'm a bit on the fence about this.. the addon handles backgrounds, and surely now it will be able to handle css variables, but still people can't use it to switch themes if not via css variables. This could confuse a few users that are using emotion/styled components or other solution that doesn't rely on css variables. What do you think @shilman ?

Reply:
Also mind that there's a cypress test in `cypress/generated/addon-backgrounds.spec.ts` which will fail because of this title change.

Reply:
@yannbf I agree that this is still primarily an addon for changing backgrounds. Maybe the caption should be:

```
"Change the background or CSS of the preview"
```